### ISSUE TITLE
LHS gets sluggish when many selected

### ISSUE THREAD

**Issue by phraenquex** [m2ms/fragalysis-frontend#768]
* Select check-boxes massively sluggish, especially with lots of hits displayed
* Tooltip with large view of molecule sluggish and temperamental (Maybe need a large/small view toggle?  Epic7, possibly.)

It's a problem because it gets hard to assign tags - i.e. do science with the tags.

**Comment by rsanchezgarc** [comment]
I am still getting some problems with the molecule view tooltip.
- Sometimes it does not pop up if the molecule is at the end of the lane
- Sometimes it does not pop up when another tooltip is being visualized.

https://user-images.githubusercontent.com/27857332/151966588-f7e8f5e5-4b16-4483-b262-d8273de95696.mp4

**Comment by phraenquex** [comment]
Yes, I noticed this too this morning - it's quite subtle when you look at individual things, but becomes really apparent if you hovever over many in quick succession, e.g. by moving the mouse down the LHS.  And since that's what it's meant for, it's debilitating.

**Comment by phraenquex** [comment]
@rsanchezgarc, my gibhub says "Video can't be played because the file is corrupt".  (Is it a firefox thing?)

**Comment by rsanchezgarc** [comment]
@phraenquex , Works in Chrome so it should be Firefox stuff
